chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/SchedulerManager.java

```diff
@@ -286,7 +286,7 @@ public ChplJob updateJob(ChplJob job) throws SchedulerException {
     }
 
     @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_ONC')")
-    public void retireAcb(String acb) throws SchedulerException, ValidationException {
+    public void retireAcb(String acb) throws SchedulerException, ValidationException, MessagingException {
```

Comment:
Is this method used? When?

Reply:
It is called from the CertificationBodyManager when an ACB is retired.